account_invoice_export/tests/test_export_invoice.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import SingleTransactionCase
+
+
+class TestExportAcountInvoice(SingleTransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.send_exception = cls.env.ref(
+            "account_invoice_export.mail_activity_transmit_warning"
+        )
+        cls.transmit_method = cls.env["transmit.method"].create(
+            {
+                "name": "HttpPost",
+                "code": "httppost",
+                "customer_ok": True,
+                "send_through_http": True,
+                "destination_url": "https://somewhere.com/post",
```

Comment:
Better to use `example.com` for test purpose

Reply:
Changed